File: app/src/main/java/com/simon_eye/cardsscoring/RoundScores.java
package com.simon_eye.cardsscoring;


/**
 * {@link RoundScores} represents a single Android platform release.
 * Each object has 3 properties: name, version number, and image resource ID.
 */

public class RoundScores {

    private int[] mPlayerScores;
    private String mRoundNumber;


    public RoundScores(int[] PlayerScores, int RoundNumber)
    {

        mPlayerScores = new int[PlayerScores.length];
        for (int i = 0; i < mPlayerScores.length; i++) {
            mPlayerScores[i] = PlayerScores[i];
        }
        mRoundNumber = String.valueOf(RoundNumber);
    }
    /**
     * Get the Round Number
     */
    public String getRoundNumber() {
        return mRoundNumber;
    }

    public String getPlayerScore(int i) {
        return String.valueOf(mPlayerScores[i]);
    }

    public int getNumOfPlayers() {
        return mPlayerScores.length;
    }
}

File: app/src/main/java/com/simon_eye/cardsScoring/RoundScores.kt
package com.simon_eye.cardsScoring


/**
 * [RoundScores] represents a single Android platform release.
 * Each object has 3 properties: name, version number, and image resource ID.
 */

class RoundScores(PlayerScores: IntArray, RoundNumber: Int) {

    private val mPlayerScores: IntArray
    /**
     * Get the Round Number
     */
    val roundNumber: String

    val numOfPlayers: Int
        get() = mPlayerScores.size


    init {

        mPlayerScores = IntArray(PlayerScores.size)
        for (i in mPlayerScores.indices) {
            mPlayerScores[i] = PlayerScores[i]
        }
        roundNumber = RoundNumber.toString()
    }

    fun getPlayerScore(i: Int): String {
        return mPlayerScores[i].toString()
    }
}
